charts/oes/values.yaml

```diff
@@ -572,7 +582,7 @@ spinnaker:
       fiat.yml:
         artifactId: docker.io/opsmxdev/ubi8-oes-fiat:1.11.0
       front50.yml:
-        artifactId: docker.io/opsmxdev/ubi8-oes-front50:0.24.0
+        artifactId: docker.io/opsmxdev/ubi8-oes-front50:0.25.1-opa
```

Comment:
Can you move all these spinnaker images to Quay and change these artifactIds so that images will be fetched from Quay?

Reply:
Done